src/js/App/RootApp.js

```diff
@@ -12,8 +13,30 @@ const RootApp = ({ activeApp, activeLocation, appId, config, pageAction, pageObj
   const isGlobalFilterEnabled =
     (!globalFilterHidden && activeLocation === 'insights') || Boolean(localStorage.getItem('chrome:experimental:global-filter'));
   const scalprum = useScalprum(config);
+  const remoteModule = useSelector(({ chrome }) => chrome?.activeSection?.module);
+  const insightsContentRef = useRef(null);
+  const legacyMainContainerRef = useRef(null);
+  useEffect(() => {
+    const contentElement = document.getElementById('root');
+    if (contentElement) {
+      legacyMainContainerRef.current = contentElement;
+    }
+    if (!remoteModule) {
+      insightsContentRef.current.appendChild(contentElement);
```

Comment:
I'd log the usage of this in Sentry, we want to ping apps that are not using new chrome-2.0 and tell them to use it. Not needed for now, but nice to have for future.

Reply:
We can add it later. I think we might hit sentry limits very soon if we log this.